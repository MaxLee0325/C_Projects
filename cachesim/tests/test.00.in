1
16
65536
L1 64 1 64
10
S 22 1234
L 22
S 48 3456
L 48
S 70 4567
L 70
S 4118 5678
L 4118
L 22
L 4118
