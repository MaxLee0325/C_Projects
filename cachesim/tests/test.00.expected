Level 1 cache: sets: 64, lines 1, block size: 64
Cache L1 miss @ 0x0016
Stored word: 0x4d2
Cache L1 hit @ 0x0016
Loaded word: 0x4d2
Cache L1 hit @ 0x0030
Stored word: 0xd80
Cache L1 hit @ 0x0030
Loaded word: 0xd80
Cache L1 miss @ 0x0046
Stored word: 0x11d7
Cache L1 hit @ 0x0046
Loaded word: 0x11d7
Cache L1 miss @ 0x1016
Stored word: 0x162e
Cache L1 hit @ 0x1016
Loaded word: 0x162e
Cache L1 miss @ 0x0016
Loaded word: 0x4d2
Cache L1 miss @ 0x1016
Loaded word: 0x162e
Cache: L1, hits: 5, misses: 5
